demo/api/src/config/environment-variables.ts

```diff
@@ -95,4 +95,16 @@ export class EnvironmentVariables {
     @ValidateIf((v) => v.DAM_STORAGE_DRIVER === "s3")
     @IsString()
     S3_BUCKET: string;
+
+    @IsString()
+    @IsOptional()
+    CONTENT_GENERATION_API_URL?: string;
+
+    @IsString()
+    @IsOptional()
+    CONTENT_GENERATION_API_KEY?: string;
+
+    @IsString()
+    @IsOptional()
+    CONTENT_GENERATION_DEPLOYMENT_ID?: string;
```

Comment:
those are for the open ai api, the env name should also reflect that imho. And we even use a specific `@azure/openai` package, does that mean it's azure specific and only works in azure?

Reply:
Yes it only works for the Azure OpenAI API. That's our default setup and that's why we put the helper service in the library.

I changed the names of the env vars and the service to reflect this connection